packages/react/index.js

```diff
@@ -43,9 +43,10 @@ module.exports = (opts = {}) => neutrino => {
             {
               // Enable development helpers both in development and testing.
               development: process.env.NODE_ENV !== 'production',
-              // Use the native built-in instead of polyfilling.
-              // Don't confuse this with `@babel/preset-env`'s `useBuiltIns` option, they are different.
-              useBuiltIns: true,
```

Comment:
We need to remove `useBuiltIns` to use the useSpread option?

Reply:
Nevermind, just saw that it only accepts only accepts useBuiltIns or useSpread in  https://github.com/babel/babel/pull/10572/files#diff-150912f9d993ec47518d06f0b1f877bcR182.

Reply:
Yeah sorry meant to mention that in the PR description.